test.env

```diff
@@ -56,3 +56,12 @@
 # export ARTIFACT_DIR=.artifacts
 # export ENABLE_RBAC=false
 # export ENABLE_PROFILING=false
+
+export DURATION=5m
+export PRE_LOAD_DB=true
+export SCENARIO=mvp
+export USE_PR_BRANCH=true
+export WAIT_FOR_SEARCH_INDEX=true
```

Comment:
@jhutar that should stay false until https://issues.redhat.com/browse/RHIDP-2247 is finished

Reply:
Thank you, fixed.